src/object/C4Id.h

```diff
@@ -34,13 +34,13 @@ class C4ID
 	static NamesList names;
 	void assign(const std::string &s);
 	template<size_t N>
-	explicit C4ID(const char (&s)[N]) { assign(s); }
+	explicit C4ID(const char (&s)[N]) { assign(s); } // TODO: v is not assigned
```

Comment:
It is assigned, either in https://github.com/openclonk/openclonk/blob/53fccd01d1ee9f77c14ac6c96b6065b01a0b9da9/src/object/C4Id.cpp#L45 or in https://github.com/openclonk/openclonk/blob/53fccd01d1ee9f77c14ac6c96b6065b01a0b9da9/src/object/C4Id.cpp#L49

Reply:
Okay, maybe the comment was misleading. The C++ code analyzer in Eclipse complains that "Member 'v' was not initialized in this constructor"